src/App.css

```diff
@@ -1,7 +1,12 @@
 .active-link {
-  color: black;
+  color: rgb(24, 24, 24);
 }
 
```

Comment:
- [ ] **Required:**

_**I'll leave my comment here because there's been no change in this PR on the file.**_

As stated earlier by the previous reviewer, the essence of this milestone is to ensure that you can follow a given high-fidelity design.

Having said that Kindly ensure your design matches the template

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/179176243-a499a09c-8731-4755-9b8a-2f33fa4dd5e7.png) | ![image](https://user-images.githubusercontent.com/27709832/179176514-7ef59710-ac97-443e-9b0e-10801865d6be.png)